pkg/blobby/testutil/test_harness.go

```diff
@@ -138,33 +155,58 @@ func (o GetOp) String() string {
 }
 
 func (o GetOp) Run(t *testing.T, ctx context.Context) error {
-	expected, _, err := o.h.model.Get(ctx, o.k)
+	val, stats, err := o.h.sut.Get(ctx, o.k)
 	if err != nil {
-		return fmt.Errorf("fakeBlobby get: %v", err)
+		// check if it's a NotFound error - that's expected for deleted/missing keys
+		if errors.Is(err, &api.NotFound{}) {
+			// verify model also returns not found
+			_, _, modelErr := o.h.model.Get(ctx, o.k)
+			if !errors.Is(modelErr, &api.NotFound{}) {
+				return fmt.Errorf("sut returned NotFound but model did not: sut=%v, model=%v", err, modelErr)
+			}
+			t.Logf("Get %s -> NotFound (expected)", o.k)
+			return nil
+		}
+		return fmt.Errorf("get: %v", err)
```

Comment:
flatten the block above by inverting the condition and returning early for errors other than notfound.